### PR TITLE
DOCS

### PR DESCRIPTION
**Pull request by amyleblond**

### CODE REVIEW

File: Documentation/README.md [hunk 1]
```diff
@@ -183,10 +183,36 @@ Last step. [Go on the algorithm page](https://team4.uwaterloo.ca/taform/algorith
...
+		8. Left align headings on Applicants table.
+			* Tips: Use CSS align property within the Applicants `th` CSS class.
+		9. Format Applicants table header to be the same as all other tables in the system.
+			* Tips: Identify the CSS class of the table style you are trying to replicate. Motify the Applicants table CSS class to replicate the `th` styling you want.
+		10. Add ascending/descending arrows to the Applicants table for sorting.
+			* Tips: This would have to be done through jQuery. Leverage jQuery exisitng librarys. 
+		11. More explicit error messaging for Student ID entry on the Edit Student Informating Page.
+			* Tips: Use the error messaging for Student ID that is currently used for the application.html page. Within views and on POST, there are built in validation checks for the length of the Student ID. 
+
+### Backup Database / Media Files
```
Comment (j2kan) [inline Documentation/README.md]: should this go before line 171

**Review by j2kan — APPROVED** [review]
1 comment then LGTM

**Review by smwatts — APPROVED** [review]
You did an amazing job Amy! For some reason the future improvements & backup are showing in these grey boxes when I view changes. Can we make those go away?

**Comment by smwatts** [comment]
screen shot 2018-03-12 at 2 44 30 pm

**Review by lauradobson — APPROVED** [review]

**Comment by amyleblond** [comment]
@smwatts @j2kan see changes!

**Comment by smwatts** [comment]
Formatting got weird lol
screen shot 2018-03-12 at 3 05 08 pm

File: Documentation/README.md [hunk 2]
```diff
@@ -168,6 +168,14 @@ Last step. [Go on the algorithm page](https://team4.uwaterloo.ca/taform/algorith
 		* The courses that did not get matched
 		* The final matching result between courses and students
 
+### Backup Database / Media Files
+* Backup
```
Comment (smwatts) [inline Documentation/README.md:172]: Remove astricks I think

**Pull request merged**